### DISCUSSION TITLE
µCNC support for GrblHAL Keypad

### DISCUSSION THREAD

**Discussion by Paciente8159** [Expatria-Technologies/RT_Jog_Controller#13] [General]
Hi there,

Great project.

I'm the maker of [µCNC](https://github.com/Paciente8159/uCNC). Recently I've added support for GrblHAL keypad specifications compatible hardware as an [additional module](https://github.com/Paciente8159/uCNC-modules/tree/master/grblhal_keypad).

Thanks

**Comment by andrewmarles** [comment]
This looks great, I have been following your work for a while.  I really like your project.  I very much want to get it ported over to our boards and this is extra incentive.  Thank you for your efforts.

**Reply by Paciente8159** [reply]
Thanks. If you get to try it, let me know if you need help with the configurations.

I left an additional entry point in the keypad module to allow expanding additional keycodes and run custom code.

I'm planning on adding additional board support via the web builder, and have a list of pre-build configured boards for download there.
This will probably speedup board coverage in the near future.
